<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Language</class>
 <widget class="QDialog" name="Language">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Ubuntu</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: black;</string>
  </property>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>190</y>
     <width>771</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>60</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>CHOOSE LANGUAGE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBG">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>400</y>
     <width>351</width>
     <height>231</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/new/prefix1/icons/bg.png</normaloff>:/new/prefix1/icons/bg.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>360</width>
     <height>270</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEN">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>400</y>
     <width>351</width>
     <height>231</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/new/prefix1/icons/en.jpg</normaloff>:/new/prefix1/icons/en.jpg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>360</width>
     <height>270</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>60</y>
     <width>601</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>60</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>ИЗБЕРЕТЕ ЕЗИК</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Icons.qrc"/>
 </resources>
 <connections/>
</ui>
